x-pack/plugins/canvas/public/expression_types/arg_types/series_style/extended_template.js

```diff
@@ -37,13 +38,29 @@ export const ExtendedTemplate = props => {
     { value: 5, text: '5' },
   ];
 
-  const labelOptions = [{ value: '', text: 'Select Series' }];
+  const labelOptions = [
+    {
+      value: '',
+      text: intl.formatMessage({
```

Comment:
please check it works for another locale. we have a issue with translating constants. If it does not work for another locale, use function instead